public_html/assets/css/nf-core-dark.css

```diff
@@ -150,6 +150,19 @@ blockquote a {
 .table-bordered th{
   border-color: #454d55;
 }
+.table tr.table-success,
+.table .table-success>td{
```

Comment:
Is this needed? https://getbootstrap.com/docs/4.6/content/tables/#contextual-classes

Reply:
This is for the dark mode, where `.table-success` looks very outwashed.

Reply:
ahh 👍🏻